tests/csapi/apidoc_room_forget_test.go

```diff
@@ -73,19 +72,18 @@ func TestRoomForget(t *testing.T) {
 			alice.MustDoFunc(t, "POST", []string{"_matrix", "client", "r0", "rooms", roomID, "forget"})
 			time.Sleep(50 * time.Millisecond)
```

Comment:
Please remove this. You can probably use a sentinel event to catch this. Do the following:
- Alice forgets the room
- Bob sends a message in roomID
- Sync until you see the message on Bob's sync stream
- Then sync Alice and check that the room isn't there.

This is called the sentinel method because you're using a separate event to act as a sentinel to detect when something previous has happened. This only reliably works when there is a single queue handling sync data, but for the case of Dendrite this should work.